corehq/form_processor/backends/sql/dbaccessors.py

```diff
@@ -584,16 +584,16 @@ def soft_delete_forms(domain, form_ids, deletion_date=None, deletion_id=None):
 
         return affected_count
 
-    @staticmethod
     @transaction.atomic
-    def _archive_unarchive_form(form, user_id, archive):
+    def send_to_kafka(self, form, user_id, archive):
```

Comment:
should undo this rename since the previous name is more correct

Reply:
https://github.com/dimagi/commcare-hq/pull/22470/commits/41f1f3a3932c312fe24b6bda81c9e8a7be828e10